draft-ietf-modpod-group-processes.md

```diff
@@ -306,10 +306,11 @@ comity of a forum will be attempted.
 The moderator team is responsible to the IESG.  The IESG
 MAY create or designate a forum to facilitate discussion about
 moderation, and refer interested parties to that forum.  All actions
-taken by the moderator team SHALL be reported to the IESG.  All
+taken by the moderator team SHALL be reported to the IESG,
+as well as to those against whom those actions are directed.  All
 bans longer than fourteen (14) days SHALL be reported to the forum in
-which the person was banned, as well as to that person, and in the
-case of a broad ban, to the community in a manner decided by the IESG.
+which the person was banned, and in the case of a broad ban, to the
```

Comment:
Suggest to replace `ban` by `moderation action`, as its more general.

What is the definition of a `broad ban`?

Reply:
In which case would the moderators take a "pre-approval to post" action?

Reply:
Heh :-) I have the exact opposite reaction. IESG yes, since the moderators are overseen by them, WG chairs no, since they aren't.

Reply:
Oversight doesn't mean knowing everything.  It means knowing what the overseer needs to know.  That might be an aggregate report.

Reply:
When an appeal comes about a moderation action taken that the IESG hasn't been informed of (yet), we're running into issues.

Reply:
I would think the responsible AD would have gotten it first, and if that person hasn't done their homework, that's a problem on its own.  But I actually agree with the sentiment of your original draft- let the moderators do their job, without having to constantly email bomb the IESG.  If someone steps WAAAY over the line, then we set the $BigDeal bit, and that requires more visibility.

Reply:
Coming back to your top level issue, i just realized that we have meandered off the topic of this PR, which is about informing the offender.

Reply:
Fair point, esp, since we're also already saying `All taken by the moderator team SHALL be reported to the IESG`.

I'll open another issue on the use of `ban`.